service-app/internal/api/handler.go

```diff
@@ -159,6 +157,7 @@ func (c *IndexController) IngestHandler(w http.ResponseWriter, r *http.Request)
 			attchResp, decodedXML, docErr := service.AttachDocuments(ctx, scannedCaseResponse)
 			if docErr != nil {
 				c.logger.Error("Failed to attach document", map[string]interface{}{
+					"AWS Trace ID":  reqID,
```

Comment:
[To match other services](https://docs.opg.service.justice.gov.uk/documentation/adrs/adr-009.html#reserved-fields) (and so we can query CloudWatch easily across them), could this key be `trace_id`?

Reply:
Also, for consistency I'll match other property keys with the same formatting.

Reply:
https://github.com/ministryofjustice/opg-scanning/pull/39/commits/e0d09cf97bb22866990a54dae48a5bb52e6551d5